packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockContextMenuItem.tsx

```diff
@@ -33,9 +40,11 @@ export const BlockContextMenuItem = forwardRef<
   const localRef = useRef<HTMLLIElement>(null);
   const menuItemRef = (ref ?? localRef) as RefObject<HTMLLIElement>;
```

Comment:
`ref` here could be a function in which case your attempt to access it in your effect breaks. I would use a tool to combine refs instead. https://github.com/gregberge/react-merge-refs

Reply:
True. Didn't consider that. Added a todo comment to handle that when working on nested menus